blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/ScheduleContext.java

```diff
@@ -481,7 +480,12 @@ public long read() {
         return results;
     }
 
-    private void registerMBean() {
+    private boolean registered = false;
```

Comment:
My only concern was with preceding the variable name with "is" which is typical of naming boolean fields in java.  I am fine with the rest of it.